src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/inverse.hpp

```diff
@@ -13,12 +13,12 @@
 namespace ov {
 namespace test {
 
-using InverseTestParams = typename std::tuple<ov::Shape,          // input shape
-                                              ov::element::Type,  // element type
-                                              bool,               // adjoint
-                                              bool,               // test_static
-                                              unsigned int,       // seed
-                                              std::string         // device_name
+using InverseTestParams = typename std::tuple<std::vector<InputShape>,  // input shape
+                                              ov::element::Type,        // element type
+                                              bool,                     // adjoint
+                                              bool,                     // test_static
```

Comment:
`test_static` - is excessive. Just need to pass static or dynamic shapes.

Reply:
Removed!